src/core/generate.test.ts

```diff
@@ -109,13 +140,13 @@ describe("generate", () => {
     });
 
     it("should generate the integration tests", () => {
-      expect(getIntegrationTestFile("./superhero", "superhero.zod"))
+      expect(getIntegrationTestFile(inputPath, outputFile))
         .toMatchInlineSnapshot(`
         "// Generated by ts-to-zod
         import { z } from \\"zod\\";
 
-        import * as spec from \\"./superhero\\";
-        import * as generated from \\"superhero.zod\\";
+        import * as spec from \\"${inputPath}\\";
+        import * as generated from \\"${outputFile}\\";
```

Comment:
I do prefer hardcode path here since this is a test, so it’s easier to see if the path is actually properly generated

Reply:
I agree with you, I left the computed value here for dev convenience since the [temp](https://www.npmjs.com/package/temp) directory in which we're placing the test generated files is being generated randomly.
we do have [a function](https://github.com/fabien0102/ts-to-zod/blob/e16953c84723b48115c4c619727e31aa4b0efd38/src/utils/nodeFileResolution.ts#L5-L30) that translates between absolute and relative paths so could also use that to generate the relative import path, I'll look into this later.

Reply:
Whatever solution I could think of was overkill if compared to the benefits of having a fixed string for the path.